lambdas/src/apis/collections/controllers/wearables.ts

```diff
@@ -22,29 +23,41 @@ export async function getWearablesByOwnerEndpoint(
   theGraphClient: TheGraphClient,
   req: Request,
   res: Response
-) {
+): Promise<void> {
   // Method: GET
-  // Path: /wearables-by-owner/:owner
+  // Path: /wearables-by-owner/:owner?collectionId={string}
```

Comment:
Should we update the docs? Also, can you add a comment here explaining the behaviour when no param is sent